src/WebJobs.Script/Description/FunctionDescriptor.cs

```diff
@@ -38,5 +39,24 @@ public FunctionDescriptor(string name, IFunctionInvoker invoker, FunctionMetadat
         public IFunctionInvoker Invoker { get; private set; }
 
         public FunctionMetadata Metadata { get; private set; }
+
+        public Collection<FunctionBinding> InputBindings { get; set; }
+
+        public Collection<FunctionBinding> OutputBindings { get; set; }
```

Comment:
1. That doesn't use OutputBindings. 
2. Is this.InputBindings  == this.Metadata.InputBindings


---
In reply to: [112280065](https://github.com/Azure/azure-webjobs-sdk-script/pull/1433#discussion_r112280065) [](ancestors = 112280065)